packages/global-styles/src/hooks/use-font-pairing-variations.ts

```diff
@@ -2,7 +2,11 @@ import { useQuery } from '@tanstack/react-query';
 import wpcomRequest from 'wpcom-proxy-request';
 import type { GlobalStylesObject } from '../types';
 
-const useFontPairingVariations = ( siteId: number | string, stylesheet: string ) => {
+const useFontPairingVariations = (
+	siteId: number | string,
+	stylesheet: string,
+	disabled = false
```

Comment:
As someone that doesn't have context for 5 months, reading the caller of this function is a bit confusing.

We need to be careful adding positional param to this as this lives in global packages, not just ours (onboarding / assembler steps).

What if we introduce the last param as options object? Something like this:

```
const useFontPairingVariations = (
	siteId: number | string,
	stylesheet: string,
	{ enabled = true }: { enabled: boolean }
)

...

enabled: !! siteId && !! stylesheet && enabled
```

then the caller is more readable:

```
const fontVariations = useFontPairingVariations(
		siteId,
		stylesheet,
		{ enabled: !!preselectedFontVariationTitle }
	);
```
```

Reply:
Addressed via [cafe67d](https://github.com/Automattic/wp-calypso/pull/78313/commits/cafe67d17c171461b628fe0f86d046463a71dc1e)